### ISSUE TITLE
Using Synology NAS, cannot see sub-folders and files

### ISSUE THREAD

**Issue by CTJohnK** [jlesage/docker-avidemux#5]
I'm new to Docker, so perhaps I have misconfigured something, but I access Avidemux and try to open sub-folders or files, nothing appears in the UI dialog.

I have the system configured -

User: '1000' with r/w access to the share 
Group: 1000 with r/w access to the share

Added Folder mapping from share to '/storage/recordings'

Explicitly set Local Ports to Container Ports.

- tried changing 'USER_ID' and 'GROUP_ID' to existing users and groups
- tried changing 'APP_USER' as default 'app' to '1000' or an existing user

Can see 'recordings' folder under 'storage' in UI, but no sub-folders or files appear.

Any idea what I have missed?

Thank you for your help in advance.

**Comment by jlesage** [comment]
Looks like a permission issue... Can you double check (with `ls -l /path`) in a terminal that your share has proper ownership/permissions ?

**Comment by CTJohnK** [comment]
ls -l /storage/recordings  returns the screenshot.
I have user 1000 and group 1000 having read/write access to the original mapped folder.

![Screenshot_20201230-224450](https://user-images.githubusercontent.com/69587153/103393474-c6a15380-4af0-11eb-9261-47427a4d9535.png)

**Comment by CTJohnK** [comment]
![Screenshot_20201230-225337](https://user-images.githubusercontent.com/69587153/103393669-ed13be80-4af1-11eb-95a3-ce1a4f313ee7.png)

**Comment by CTJohnK** [comment]
Hopefully this set of screenshots makes better sense -
This is the revised volume mapping to the 'root' multimedia share:
![Capture](https://user-images.githubusercontent.com/69587153/103411291-29204100-4b3d-11eb-99f0-9f9a78489c4d.JPG)


Here is the user 1000 config:
![Capture5](https://user-images.githubusercontent.com/69587153/103411389-b2d00e80-4b3d-11eb-9431-aa87364ad6a6.JPG)

![Capture6](https://user-images.githubusercontent.com/69587153/103411496-4f92ac00-4b3e-11eb-8668-d57a45369855.JPG)

1000 Group setting:
![Capture7](https://user-images.githubusercontent.com/69587153/103411438-f0cd3280-4b3d-11eb-9c10-7f480e7a53dc.JPG)


"ls -l /storage"  (interesting note, the old mapping directly to 'recordings' still appears even though it was removed from the volume mapping)
![Capture1](https://user-images.githubusercontent.com/69587153/103411132-7354f280-4b3c-11eb-9ee2-f19e4f813191.JPG)

"ls -l /storage/multimedia" (/multimedia is the 'root' of the share and the user 1000 and group 1000 have R/W access to 'multimedia')
![Capture2](https://user-images.githubusercontent.com/69587153/103411171-9a132900-4b3c-11eb-83dd-2b11d93746a1.JPG)

"ls -l /storage/multimedia/recordings"
![Capture3](https://user-images.githubusercontent.com/69587153/103411219-d3e42f80-4b3c-11eb-890e-9b92efac8251.JPG)

"ls -l /storage/multimedia/recordings/tv"
![Capture4](https://user-images.githubusercontent.com/69587153/103411226-e0688800-4b3c-11eb-8925-8130def48d16.JPG)



What Avidemux shows in Folders:
Initial folder screen:
![Capture8](https://user-images.githubusercontent.com/69587153/103411595-d8a9e300-4b3e-11eb-82da-9b39b3e10c7a.JPG)

Drilling into 'storage':
![Capture9](https://user-images.githubusercontent.com/69587153/103411625-ecede000-4b3e-11eb-92bc-70db4d603613.JPG)

Drilling into 'storage/multimedia': 
![CaptureA](https://user-images.githubusercontent.com/69587153/103411650-13ac1680-4b3f-11eb-9f50-82ee013d63e3.JPG)

**Comment by CTJohnK** [comment]
One other note, this behavior occurs with Chrome, Firefox or VNC.

**Comment by jlesage** [comment]
The `multimedia` folder is owned by root/root and there is permission on it...

Also, from what I understand, the user `1000` is the name of the user, not its numerical id.  To remove confusion, I would suggest that you use a real name, like `cont_app` or something like that.  Then use https://github.com/jlesage/docker-avidemux#usergroup-ids to find out its numerical ids and use them in `USER_ID` and `GROUP_ID` environment variables of the container.

**Comment by CTJohnK** [comment]
This seems to be the issue.  When I run the 'ID' command ('id 1000', 'id admin' or 'id root') only 'id root' returns a value, the others return "id: unknown user admin" as an example for 'id admin'".

I'd rather not run the container with the 'root' user ID of '0', but I cannot get the container to report any other UID.

Thank you for your response and I hope you're enjoying the holidays.

**Comment by jlesage** [comment]
Did you try to rename your user ?

**Comment by CTJohnK** [comment]
Ok, I figured this out.  Turns out I could not execute the 'ID' command from the container's Terminal session available in Synology's DSM.  I needed to actually SSH into the NAS to get the ID.

'Fun fact' if I set the Environment up with USER_ID = 1000, GROUP_ID = 1000 and APP_USER = app, the Terminal command from within the Container (id app) returned '1000' for the user and group.  If I changed the USER_ID and GROUP_ID to 1024, cleared, restarted and ran 'id app' again, it returned '1024' for both.

So, I SSHed into the NAS and ran 'id app' and got a uid of 1035 and gid of 100.  Plugged those into the Environment variables and changed user name '1000' to 'app' - I'm able to get in.

Thank you for your help, sorry I was a bit 'slow'.
